packages/flutter_test/lib/src/controller.dart

```diff
@@ -24,6 +24,17 @@ const double kDragSlopDefault = 20.0;
 
 const String _defaultPlatform = kIsWeb ? 'web' : 'android';
 
+// This is to prevent `SemanticsController.find` from shadowing `finders.dart#find`.
+const CommonFinders _find = find;
```

Comment:
It would probably be more idiomatic to import it as `import 'finders.dart' as finders;` and then use it as `finders.find`.

Reply:
That was my first idea as well, but it resulted in a TON of unrelated changes in this file to enable four calls to `finders.find` in `SemanticsController`.

Do you think it's better to bite the bullet and just do those unrelated changes?

Reply:
I think we should just bite the bullet. BUT: Maybe we can do this in a separate PR and submit that one first to keep this PR reviewable?

Reply:
Good call. PR is available as #136423 if you could take a look!

Reply:
Alright, should be handled now. Did some minor reformatting on some parameter lists that were getting a bit too long, so a quick readability glance may be a good idea, but otherwise should be good!